lib/kino_maplibre/map_cell.ex

```diff
@@ -511,4 +516,15 @@ defmodule KinoMapLibre.MapCell do
     valid_lat? = lat >= -90 and lat <= 90
     if valid_lng? and valid_lat?, do: {lng, lat}
   end
+
+  defp normalize_geocode_id(query) do
+    if Regex.match?(~r/^[\d-]*$/, query),
+      do: "postalcode_#{String.replace(query, ~r/\D+/, "")}",
+      else:
+        query
+        |> String.downcase()
+        |> String.normalize(:nfd)
+        |> String.replace(~r/[^a-zA-Z\s]/u, "")
+        |> String.replace(~r/\W+/, "_")
```

Comment:
```suggestion
    if Regex.match?(~r/^[\d-]*$/, query) do
      "postalcode_#{String.replace(query, ~r/\D+/, "")}"
    else
      query
      |> String.downcase()
      |> String.normalize(:nfd)
      |> String.replace(~r/[^a-zA-Z\s]/u, "")
      |> String.replace(~r/\W+/, "_")
    end
```

Subjective, but especially for multiline expressions the block syntax seems cleaner :)